sum-of-multiples/example.js

```diff
@@ -0,0 +1,18 @@
+const Sum = (factors = [3, 5]) => {
+  let self = this instanceof Sum ? this : Object.getPrototypeOf(Sum);
```

Comment:
I agree it's a little weird.  

Like with #153 I would rather export a class, but that's not what the test wants.  How would you have written `example.js`?